src/FileList/FileExtInfoLoader.cc

```diff
@@ -1289,22 +1303,32 @@ void FileExtInfoLoader::AddBeamEntry(CARTA::FileInfoExtended& extended_info, con
     }
 }
 
-void FileExtInfoLoader::ConvertUnitToCasacore(casacore::String& unit) {
-    // Returns casacore Unit string or input string if unknown (fail with error later)
-    if (casacore::UnitVal::check(unit)) {
+void FileExtInfoLoader::NormalizeHeaderUnit(casacore::String& unit) {
```

Comment:
There may be some duplication between this function and the one in [#1269](https://github.com/CARTAvis/carta-backend/pull/1269). Would it make sense to consolidate these into a shared function?

Reply:
I did some more testing with units, and moved the function to Util/Casacore.h after simplifying it.  I am using two casacore functions (1) casacore::UnitVal::check and (2) casacore::UnitMap::fromFITS converts to casacore unit string, which also seems to conform to the FITS standard (lowercase or capitalized).

Unfortunately, capitalized units may pass the check and fail later on, i.e. check("Jy/Beam")=true (Flux Density issue #1187 PR #1269), check("JY/BEAM")=true, check("jy/beam")=false, check("Degrees")=false (issue  #1265).

If check returns true, the unit is converted to casacore, i.e. fromFITS("Jy/Beam")="Jy/Beam" (still fails for Flux Density), fromFITS("JY/BEAM")="Jy/beam" (ok), fromFITS("DEGREES")="deg" (ok).

So I think the safest path is to convert the unit to uppercase, check it, then convert it fromFITS for the proper form. If the check fails, leave the units as unknown to casacore.

In this PR, the units are only converted for the file info headers (used by the frontend), but not in the image itself (used by the backend).

Reply:
Side note: for Flux Density, casacore::ImageStatistics checks that the image unit string contains "/beam", which is why "/Beam" fails.  Somewhere the header "/BEAM" must be converted to the unit "/beam" (perhaps using the fromFITS function?) since it passes this check.